multiversx_sdk_core/transaction_factories/delegation_factory.py

```diff
@@ -16,23 +16,15 @@
 
 class IConfig(Protocol):
     chain_id: IChainID
-    min_gas_price: IGasPrice
     min_gas_limit: IGasLimit
     gas_limit_per_byte: IGasLimit
```

Comment:
why do you initialize this properties, as you have them already initialized on transaction_factory_config?

Reply:
Good point, will remove them in a separate PR (next one).

Reply:
Fixed in https://github.com/multiversx/mx-sdk-py-core/pull/29.